test/conformance/ingress/rewrite.go

```diff
@@ -34,7 +34,7 @@ func TestRewriteHost(t *testing.T) {
 	name, port, _ := CreateRuntimeService(ctx, t, clients, networking.ServicePortNameHTTP1)
 
 	privateServiceName := test.ObjectNameForTest(t)
-	privateHostName := privateServiceName + "." + test.ServingNamespace + ".svc.cluster.local"
+	privateHostName := privateServiceName + "." + test.ServingNamespace + ".svc." + test.NetworkingFlags.ClusterSuffix
```

Comment:
Can we use https://github.com/knative/pkg/blob/3a2ae6db70971afbcaf1e1c3826cbc9a5d6c718e/network/domain.go#L40 instead?

Reply:
It is not possible unfortunately.
`GetServiceHostname()` reads `/etc/hosts` so it works within a pod like activator code, queue-proxy code, etc. But here is a test client so it seems to read the `/etc/hosts` in the host machine which kicked test.  

I wondered that we can drop the `--cluster-suffix` option, but I understand why we cannot drop it :sweat_smile: